scout/server/blueprints/cases/controllers.py

```diff
@@ -680,6 +680,7 @@ def _append_evaluated_variant_by_type(
                             ]
                         )
                     )
+                    print(f"var[bayesian_acmg] is -->{var_obj['bayesian_acmg']}")
```

Comment:
The right bayesian classification is returned for [this variant](https://scout-stage.scilifelab.se/cust002/F0017133/f40feec9c34fa49398910972e3c1ae8c):

<img width="1101" alt="image" src="https://github.com/user-attachments/assets/7e8b21e6-2a6b-427b-a630-4b05314c2a36" />